<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwapCrypto</class>
 <widget class="QDialog" name="SwapCrypto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(56, 57, 57, 196);</string>
  </property>
  <widget class="QPushButton" name="inputButton_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(75, 155, 255);</string>
   </property>
   <property name="text">
    <string>Swap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Futura</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(75, 155, 255);</string>
      </property>
      <property name="text">
       <string>Swap Crypto</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>145</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>To</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(75, 155, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_0">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-color: rgb(149, 151, 151);}

QPushButton:pressed{background-color: rgb(141, 144, 144);}</string>
   </property>
   <property name="text">
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>141</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Swap</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 13pt &quot;Futura&quot;;
selection-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 13pt &quot;Futura&quot;;
selection-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Enter Amount To Swap:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Futura</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
